geomstats/datasets/sample_sdp_2d.py

```diff
@@ -0,0 +1,131 @@
+"""Helper data classes for the MDM illustration example on SPD matrices."""
+
+import geomstats.backend as gs
+from geomstats.geometry.spd_matrices import SPDMatrices
+from geomstats.geometry.special_orthogonal import SpecialOrthogonal
+
+
+class DatasetSPD_2D():
+    """
+    Sample 2D SPD dataset.
+
+    Data is of shape [n_samples * n_classes, n_features];
+    Labels are of shape [n_samples * n_classes, n_classes].
+    """
+
+    def __init__(self, n_samples=100, n_features=2, n_classes=3):
+        self.n_samples = n_samples
+        self.n_features = n_features
+        self.n_classes = n_classes
+        self.data_helper = DataHelper()
+
+    def generate_sample_dataset(self):
+        """
+        Generate the dataset.
+
+        :return:
+        X: array-like, shape = [n_samples * n_classes, n_features]: data
+        Y: array-like, shape = [n_samples * n_classes, n_classes]: labels
+        """
+        X, Y = self.setup_data()
+        X, Y = self.data_helper.shuffle(X, Y)
+        return X, Y
+
+    def setup_data(self):
+        """
```

Comment:
We use a different convention for docstring, see:
https://geomstats.github.io/contributing.html#documentation